lib/std/Target.zig

```diff
@@ -3069,10 +3056,7 @@ pub fn cTypeBitSize(target: Target, c_type: CType) u16 {
             },
             .longlong, .ulonglong, .double => return 64,
             .longdouble => switch (target.cpu.arch) {
-                .aarch64 => switch (target.os.tag) {
-                    .bridgeos => return 128,
-                    else => return 64,
-                },
+                .aarch64 => 64,
```

Comment:
```suggestion
                .aarch64 => return 64,
```

Reply:
I already fixed this in fact. But GitHub seems to be having a rather slow day:

![image](https://github.com/user-attachments/assets/40ed7268-6435-4905-b729-e1742efc968c)